frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/premium.embedding.unit.spec.tsx

```diff
@@ -56,6 +56,34 @@ describe("[EE, with token] embedding settings", () => {
       });
     });
 
+    describe("interactive embedding", () => {
```

Comment:
The changes are exactly the same as the `common.embedding.unit.spec.tsx` except the assertion for CORS since it'll only be enabled when `enable-embedding-sdk: true` and `embedding_sdk` flag is present.